plugins/woocommerce-admin/client/layout/controller.js

```diff
@@ -337,8 +337,8 @@ export const getPages = () => {
 			],
 			layout: {
 				header: false,
-				footer: false,
-				showNotices: false,
+				footer: true,
```

Comment:
Not an issue, but I'm curious as to why `footer` is needed? I've tried removing [it from the dom](https://github.com/woocommerce/woocommerce/blob/8a62ad18c727a73a7490dbf5dab1e3b2a01b82ff/plugins/woocommerce-admin/client/layout/footer/footer.tsx#L23C1-L29) and it seems like it has no impact.

Reply:
@ilyasfoo Good question! It's needed because `TransientNotices` renders the `SnackbarList` in the `WooFooterItem` slot. 

I've also tried to remove it and can see that it has an impact. 🤔 

https://github.com/woocommerce/woocommerce/blob/de6f77d8be5c9f9889e80c6400acc8173fbbae84/plugins/woocommerce-admin/client/layout/transient-notices/index.js#L93-L100

Reply:
Gotcha, thanks for checking!!